src/bci_build/package/__init__.py

```diff
@@ -356,7 +361,7 @@ def __post_init__(self) -> None:
 def _build_tag_prefix(os_version: OsVersion) -> str:
     if os_version == OsVersion.TUMBLEWEED:
         return "opensuse/bci"
-    if os_version == OsVersion.BASALT:
+    if os_version == OsVersion.SLE16_0:
         return "alp/bci"
```

Comment:
```suggestion
        return "bci"
```
or this?
```suggestion
        return "slcc/stable"
```

Reply:
my best bet is on "bci" for now, so that's how I changed it.